package.json

```diff
@@ -5,6 +5,7 @@
 	"license": "MIT",
 	"repository": "sindresorhus/norway-postal-codes",
 	"funding": "https://github.com/sponsors/sindresorhus",
+	"types": "index.d.ts",
```

Comment:
You need to add `index.d.ts` to the `files` field too.

Reply:
Done. Also moved the `types` entry to after dependencies as recommended in the style guide